docs/api_reference/fairlearn.metrics.rst

```diff
@@ -4,4 +4,11 @@ fairlearn\.metrics package
 .. automodule:: fairlearn.metrics
     :members:
     :undoc-members:
-    :show-inheritance:
\ No newline at end of file
+    :show-inheritance:
+    
+.. versionadded:: 0.7
+        In previous versions only the ``predict`` method was used
+        implicitly.
+.. versionchanged:: 0.7
+        From version 0.7, 'predict' is deprecated and the default will
+        change to 'auto' from v0.10.
```

Comment:
Hi @amalik17 ! Thanks for the contribution! I'm not entirely sure what you're trying to express with these comments since this is referring to the entire module. Did you mean to put this on the API reference of a specific class or function?

Reply:
@amalik17 are you still interested in continuing with this PR? I'd be more than happy to point you in the right direction, answer questions, or hop on a call to help figure this out. If you're no longer interested, just let us know so that someone else can pick it up. Since we sometimes don't get responses I'd suggest we wait at least until 1/20 (two weeks from today) for a response.